### ISSUE TITLE
Ask users to email us if they can't fill out the CAPTCHA

### ISSUE THREAD

**Issue by dfabulich** [iftechfoundation/ifdb#1246]
CAPTCHAs can be difficult to fill out for actual humans, especially folks with visual impairments, or in mainland China where Google scripts fail to load.

We should have messaging around the CAPTCHA saying "if you're having trouble with this, email us at ifdbadmin@ifdb.org". A moderator can then manually activate the user's account.
